backend/tests/unit/core/clients/test_slack.py

```
@@ -0,0 +1,160 @@
+
+class TestSendConfirmationMessage:
+	"""Test send_confirmation_message method."""
+
+	@patch.object(Slack, "send_channel_block_message")
+	def test_send_confirmation_message_calls_block_message(
+		self, mock_send_block_message
+	):
+		"""Test that confirmation message calls send_channel_block_message."""
+		mock_send_block_message.return_value = ("C12345", "1234567890.123456")
+
+		slack = Slack(slack_bot_token="xoxb-test-token")
```

Ruff (0.14.8)

Possible hardcoded password assigned to argument: "slack_bot_token"

(S106)